bindata/network/network-metrics/000-ns.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    name: openshift-network-metrics
+    openshift.io/cluster-monitoring: "true"
+    openshift.io/run-level: "1"
```

Comment:
Since this isn't required for the SDN to run and the operator doesn't consider the daemonset critical, we should consider removing this and use an SCC instead (if necessary).

Reply:
It turned out no special capabilities are needed, removed the annotation.